creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -333,7 +343,8 @@ class SnapbackSM {
         removeOnComplete: SNAPBACK_QUEUE_HISTORY,
         removeOnFail: SNAPBACK_QUEUE_HISTORY
       },
-      settings
+      settings,
+      ...(!_.isEmpty(limiter) && { limiter })
```

Comment:
hahaha this is impossible to read, can we just make it more verbose instead of this one liner

Reply:
sure but could you do something like this
```
  // Initialize bull queue instance with provided name and settings
  createBullQueue(queueName, settings = {}, limiter = null) {
    const queueOptions = {
      redis: {
        port: this.nodeConfig.get('redisPort'),
        host: this.nodeConfig.get('redisHost')
      },
      defaultJobOptions: {
        // removeOnComplete is required since the completed jobs data set will grow infinitely until memory exhaustion
        removeOnComplete: SNAPBACK_QUEUE_HISTORY,
        removeOnFail: SNAPBACK_QUEUE_HISTORY
      },
      settings
    }
    if (limiter) {
      queueOptions.limiter = limiter
    }
    return new Bull(queueName, queueOptions)
  }
```

Reply:
or does this work?
```
  // Initialize bull queue instance with provided name and settings
  createBullQueue(queueName, settings = {}, limiter = null) {
    const queueOptions = {
      redis: {
        port: this.nodeConfig.get('redisPort'),
        host: this.nodeConfig.get('redisHost')
      },
      defaultJobOptions: {
        // removeOnComplete is required since the completed jobs data set will grow infinitely until memory exhaustion
        removeOnComplete: SNAPBACK_QUEUE_HISTORY,
        removeOnFail: SNAPBACK_QUEUE_HISTORY
      },
      settings,
      limiter
    }
    return new Bull(queueName, queueOptions)
  }
```

Reply:
oh I thought this was a different PR. yeah that was the other way I was thinking -- doesn't seem as clean but I guess more readable so nbd. will push a new commit on this PR soon!

Reply:
dope, sg

Reply:
changed and confirmed it works in both cases (limiter with value for local or null for staging+prod) ✅